includes/Widgets/Stories.php

```diff
@@ -71,8 +71,10 @@ public function __construct( Assets $assets ) {
 		$id_base        = 'web_stories_widget';
 		$name           = __( 'Web Stories', 'web-stories' );
 		$widget_options = [
-			'description' => __( 'Display Web Stories in Sidebar Section.', 'web-stories' ),
-			'classname'   => 'web-stories-widget',
+			'description'                 => __( 'Display Web Stories in Sidebar Section.', 'web-stories' ),
+			'classname'                   => 'web-stories-widget',
+			'customize_selective_refresh' => true,
```

Comment:
Our widget doesn't actually support selective refresh, does it? So this change seems unwanted.

Reply:
Copy and pasted it from another widget. Removed now.